.storybook/preview.tsx

```diff
@@ -74,7 +75,6 @@ const preview: Preview = {
         order: ['StyleGuide', 'Components', 'Fields', 'App Layout'],
       },
     },
-    actions: { argTypesRegex: '^on[A-Z].*' },
```

Comment:
See : https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#implicit-actions-can-not-be-used-during-rendering-for-example-in-the-play-function

And i think it was not used anyway